sonar-plugin-api/src/main/java/org/sonar/api/server/authentication/UserIdentity.java

```diff
@@ -191,4 +208,16 @@ private static void validateEmail(@Nullable String email) {
       checkArgument(email == null || email.length() <= 100, "User email size is too big (100 characters max)");
     }
   }
+
+  private enum ValidateGroupName implements Predicate<String> {
```

Comment:
Do the WS `api/users/create` and `api/users/update` use the same codebase for validating group names ?